lesson7giang/src/main/java/com/example/lesson7giang/StudentDAO.java

```diff
@@ -0,0 +1,23 @@
+package com.example.lesson7giang;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import java.util.List;
+
+@Dao
+public interface StudentDAO {
```

Comment:
Các tên viết tắt nên coi là 1 từ nên ko cần phải viết hoa hết các kí tự

https://developer.android.com/kotlin/style-guide#naming_2

```suggestion
public interface StudentDao {
```